### ISSUE TITLE
Source map error in meteor prevents chrome from loading any source maps

### ISSUE THREAD

**Issue by RealHandy** [dreyescat/react-rating#90]
I get this in the client devtools console at start when I use react-rating in my meteor project:
`DevTools failed to parse SourceMap: http://localhost:3000/packages/react-rating.js.map`

It prevents all source maps from loading, which hampers my debug efforts. Any idea why react-rating does this, when none of the other 100 npm modules cause this problem? I'm not that familiar with how source maps work, so if this makes sense to you, it'd be great to resolve it.

**Comment by dreyescat** [comment]
I don't know... Uhm... Is it there? In http://localhost:3000/packages/react-rating.js.map? This source map is generated by webpack automatically. So I don't think is an actual *parse* error but file not found error or something else.

**Comment by RealHandy** [comment]
No, it’s not. Maybe it’s webpack+meteor related? I haven’t seen another package that has this behavior with the source maps.
